README.md

```diff
@@ -1,2 +1,59 @@
-# aspace-deployment
-Deployment configuration for ArchivesSpace
+# ArchivesSpace Deployment
+
+ArchivesSpace is an archives management application. For more information, please refer to the ArchivesSpace [repository](https://github.com/archivesspace/archivesspace).
+
+### Purpose
+
+The purpose of this documentary is to document the configuration and plugins of each ArchivesSpace environement we use:
+- `development`: refers to the local development environment each developer is running on their machine.
+- `test`: refers to the remote test instance currently managed by Lyrasis. This is the instance where all testing is done.
+- `production`: refers to the ArchivesSpace production instance, which is also managed by Lyrasis.
+
+Each directory in this repository is named after one of the environments listed above. Under each directory, there are two files:
+
+- `config.rb`: contains all the custom configurations to the ArchivesSpace environment
+- `plugins.yml`: contains a list of all the plugins included in the the ArchivesSpace environment. Each plugin must have the following attributes:
+
+```yml
+plugins:
+  - name: aspace_static_plugins # matches git repository name
+    git: true
+    branch: "v0.2.1" # matches the version tag or branch in the repository
+    url: https://github.com/emory-libraries/aspace_static_plugins.git # url of the plugin's git repository
+```
+
+### Plugins
+
+At Emory Libraries, we manage all ArchivesSpace UI customizations under one plugin, [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins). We also use other plugins listed under `plugins.yml` of each environment e.g. [container_management_labels](https://github.com/emory-libraries/container_management_labels).
+
+### Deployment Process
+
+1. Create a new release for [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) if the latest release does not contain the most up to date changes, and ensure the new release has a tag number that follows semantic versioning. You can use the following [instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) to create a new release.
```

Comment:
Before this the version number needs to be updated in config.rb.

Reply:
Please refer to my comment below